sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -143,7 +143,19 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
 
   /** List of test cases to ignore, in lower cases. */
   protected def blackList: Set[String] = Set(
-    "blacklist.sql"   // Do NOT remove this one. It is here to test the blacklist functionality.
+    "blacklist.sql",   // Do NOT remove this one. It is here to test the blacklist functionality.
+    // SPARK-28885 String value is not allowed to be stored as numeric type with
+    // ANSI store assignment policy.
+    "postgreSQL/numeric.sql",
+    "postgreSQL/int2.sql",
+    "postgreSQL/int4.sql",
+    "postgreSQL/int8.sql",
+    "postgreSQL/float4.sql",
+    "postgreSQL/float8.sql",
+    // SPARK-28885 String value is not allowed to be stored as date/timestamp type with
+    // ANSI store assignment policy.
+    "postgreSQL/date.sql",
```

Comment:
@gengliangwang Sorry, I just realized recently that my changed are not tested by `date.sql` and `timestamp.sql` any more when I run: `build/sbt "sql/test-only *SQLQueryTestSuite -- -z timestamp.sql"`. Did you disable them forever or are they executed in some way by jenkins?

Reply:
How about just setting `storeAssignmentPolicy=LEGACY` for the PgSQL tests? They have a lots of INSERT queries having implicit casts from string literals to numeric values.

Reply:
@maropu I am not sure about that. 
For PgSQL, it disable inserting string values to numeric columns except for literals. Setting `storeAssignmentPolicy=LEGACY` for all the PgSQL tests seems inaccurate.

Reply:
Ur, I see...

Reply:
But, before this pr's been merged, we tested the PgSQL tests in the LEGACY mode? Is my understanding wrong? Personally, I think we need to explicitly file issues in jira if we have inaccurate tests in `pgSQL/`.

Reply:
How about changing the timestamp/date values from string literal to timestamp/date literal in those sql files, just as https://github.com/apache/spark/pull/26107/files#diff-431a4d1f056a06e853da8a60c46e9ca0R68
I am not sure about whether there is a guideline in porting the PgSQL test files. Such modifications are allowed, right?

Reply:
> I am not sure about whether there is a guideline in porting the PgSQL test files. Such modifications are allowed, right?

Yea, I think that's ok.  cc: @wangyum @dongjoon-hyun @HyukjinKwon 
If you have no time to do, I'll will check next week.

Reply:
I'm +1 for @gengliangwang 's suggestion `enabling those tests with that modification`.